agent/agents/supervisor/supervisor.go

```diff
@@ -461,6 +461,7 @@ func (s *Supervisor) startBuiltin(agentID string, builtinAgent *agentpb.SetState
 		params := &perfschema.Params{
 			DSN:                  dsn,
 			AgentID:              agentID,
+			QueryLength:          int(builtinAgent.QueryLength),
```

Comment:
Same, what is the reason to use `int`?

Reply:
@artemgavrilov 

1) It was actually unspecified integer already before.
2) Postgres query max length is 32 bits, but for example MySQL is almost infinitely. Thats why I picked int to have same data type for all agents types. Anyway because I am limiting it on API layer to 32 bits anyway maybe its probably not necessary.

Its up to you about point 2. Just let me know.

Reply:
I guess we can use `int32` everywhere just to eliminate converting and simplify code.